custom-completions/gh/gh-completions.nu

```diff
@@ -547,3 +547,24 @@ export extern "gh variable" [
     --repo(-R)      # Select another repository using the [HOST/]OWNER/REPO format
     --help          # Show help for command
 ]
+
+export def "gh pr view inlined-comments" [
+    pr?: int
+    repo?: string  # e.g. nushell/nu_scripts
+] {
+# nushell/nu_scripts/pull/1105
+    let pr = if ($pr == null) { ^gh pr view --json number | from json | get number } else { $pr }
+    let repo = if ($repo == null) {
+        ^gh repo view --json name,owner | from json | select owner.login name | rename owner name
+    } else {
+        $repo | parse '{owner}/{name}' | get 0
+    }
+
+    ( (gh api
+          -H "Accept: application/vnd.github+json"
+          -H "X-GitHub-Api-Version: 2022-11-28"
+          $"/repos/($repo.owner.)/($repo.name)/pulls/($pr)/comments")
+      | from json
+      | select user.login body diff_hunk
+      | rename user comment diff )
+}
```

Comment:
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.

Reply:
I'm not sure what this comment even means. Is this to educate the reader of what Lorem Ipsum is? Is Lorem Ipsum used in this code somewhere?

Reply:
İt was to make the screenshots on the pr description, a showcase of what happens with long comments, the column body grows and diffs are hidden behind … and when using explore scrolling to the right is needed. Sorry for the confusion.